Mathlib/RingTheory/Valuation/Integers.lean

```diff
@@ -70,17 +70,15 @@ theorem one_of_isUnit' {x : O} (hx : IsUnit x) (H : ∀ x, v (algebraMap O R x)
       (algebraMap O R).map_mul, v.map_mul]
     exact mul_le_mul_left' (H (u⁻¹ : Units O)) _
 
-variable (hv : Integers v O)
-
-theorem one_of_isUnit {x : O} (hx : IsUnit x) : v (algebraMap O R x) = 1 :=
+theorem one_of_isUnit (hv : Integers v O) {x : O} (hx : IsUnit x) : v (algebraMap O R x) = 1 :=
```

Comment:
To me (not an expert on this area), this looks like an interesting argument order. Would putting `hv` after `hx` be better?

Reply:
I agree it's best to preserve argument order where it is easy to, during these refactors.